widgetastic_manageiq/expression_editor.py

```diff
@@ -125,6 +125,11 @@ class date_relative_form_view(View):  # noqa
     SELECT_SPECIFIC = ".//a[@title='Click to change to a specific Date/Time format']"
     SELECT_RELATIVE = ".//a[@title='Click to change to a relative Date/Time format']"
 
+    # This returns the whole expression unlike EXPRESSION_TEXT which only returns
+    # the first expression
+    COMPLETE_EXPRESSION_TEXT = '//*[@id="exp_editor_div"]//p'
```

Comment:
My thought is that this should be the `EXPRESSION_TEXT` and the current `EXPRESSION_TEXT` should be renamed to `FIRST_EXPRESSION` or something similar. It is a bit confusing to have `EXPRESSION_TEXT` and `COMPLETE_EXPRESSION_TEXT`. This could, however, result in a more major refactor. What do you think?

Reply:
I agree. I was thinking to make the changes you suggested but it would've led to a bigger refactor, probably breaking a few things, which is why I took this approach. But I don't mind working on it. Let me see how I can refactor further.

Reply:
Hey @john-dupuy, looks like it wasn't that big refactor after all. Hopefully, my current changes wouldn't break other tests. The test you provided passes locally. I'm assuming you had to write a separate function( `select_expression_text` ) to select the expression because of StaleElementReferenceException, I have added a comment to that function about the same. Let me know if that is not the case.